contents/blog/eu-cloud-hosting.md

```diff
@@ -0,0 +1,89 @@
+---
+date: 2022-10-17
+title: How we built EU Cloud hosting, from idea to launch
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+categories: ["Inside PostHog", "Startups"]
+author: ["ian-vanagas"]
+featuredImage: ../images/blog/posthog-eu-blog.png
+featuredImageType: full
+---
+
+We recently launched EU Cloud, providing all the benefits of cloud-hosted PostHog with data staying in the EU. We thought it would be interesting to go over why and how we built EU Cloud hosting, from idea to launch.
+
+## Why EU Cloud hosting?
+
+PostHog provides self-hosting options. Anyone can use PostHog and manage the infrastructure entirely how they want. This allows them to be more compliant with relevant laws in their industry or region. One of the big laws requiring compliance is [GDPR](/docs/integrate/gdpr) in Europe. Any time you see one of those annoying cookie banners, you’re seeing the results of GDPR. 
+
+To be more compliant with GDPR, EU user data must be stored in the EU. Because most products online have users in the EU, this requires them to do so. Not doing so could risk legal action (if this concerns you, talk to a lawyer; we are not lawyers and this is not legal advice). 
+
+Basically, we wanted to provide the benefits of cloud management with the compliance of EU data hosting. We had many PostHog users asking for it too. This is where EU Cloud was born.
+
+## Making it happen
+
+Our team did a lot of work on the infrastructure side to make EU Cloud viable before considering building it as a feature, specifically [Team Infrastructure](/handbook/small-teams/infrastructure). 
+
+They continually improved the infrastructure to make deploying, scaling, and managing PostHog easy, fast, and reliable. These improvements lead to the work on EU Cloud, and what you’ll read below benefitted a lot from it.
+
+## Picking a region and domain
+
+Once the decision to build EU Cloud happened, our team looked at the infrastructure needed and started making key decisions. An early and important one was picking a region of AWS. They recommended and choose `eu-central-1` because:
```

Comment:
```suggestion
Once we made the decision to build EU Cloud, our team looked at the infrastructure needed and started making key decisions. An early and important one was picking a region of AWS. They recommended and choose `eu-central-1` because:
```